pyspeckit/spectrum/units.py

```diff
@@ -697,9 +704,12 @@ def as_unit(self, unit, frame=None, quiet=True, center_frequency=None,
         elif xtype_check == 'fix':
             self._fix_type()
 
-        if unit in (None,'none'):
```

Comment:
Probably, this whole method can be modified to:
`return self.to(unit, self.equivalencies() + equivalencies())`, where `unit` and `equivalencies` would be passed to this method.  However, we still need to accept `center_frequency` and `central_frequency_units`